api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -238,7 +239,7 @@ private static DbAccessModule getDbAccessModuleFromApi(
    */
   public static Timestamp deriveExpirationTimestamp(Timestamp completionTime, Long expiryDays) {
     Preconditions.checkNotNull(
-        expiryDays, "expected value for config key accessRenewal.expiryDays.expiryDays");
+        expiryDays, "expected value for config key access.renewal.expiryDays.expiryDays");
```

Comment:
I see this `.expiryDays.expiryDays`. Seems to be a mistake with copy/paste.

Reply:
Heh, nice catch